interface/orders/procedure_provider_edit.php

```diff
@@ -423,8 +431,9 @@ class='form-control' />
                     </div>
                     </div>
                     <?php //can change position of buttons by creating a class 'position-override' and adding rule text-alig:center or right as the case may be in individual stylesheets ?>
+                   <div class="mt-3">
                     <div class="form-group clearfix" id="button-container">
-                        <div class="col-sm-12 text-left position-override">
+                        <div class="col-sm-12 text-center position-override">
```

Comment:
I just noted this. Turns out the was a bit of contention of where the buttons get placed. The light theme author wanted it left and the colors author wanted it centered. This is what the position-override is there for. Thus in left they go to the left and then the colors theme uses the position-override to center it. Since I missed this on the review, I'll correct this.

Reply:
Thank you for the review, I will keep that in mind next time :)